### PR TITLE
(DOCSP-9946) Add DB prereq to Compass landing

### PR DESCRIPTION
**Pull request by zach-carr**
[JIRA](https://jira.mongodb.org/browse/DOCSP-9946)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/compass/zach.carr/DOCSP-9946/index.html#prerequisites)

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -24,6 +24,17 @@ exploring your data in a visual environment, you can also use
 :ref:`manage indexes <compass-indexes>`, and implement
 :ref:`document validation <validation>`.
 
+Prerequisites
+-------------
+
+To use |compass-short|, you must connect to an existing MongoDB 
+database. You can connect to:
+
+- A MongoDB server that you :manual:`install </installation/>` and
```
Comment (andf-mongodb) [inline source/index.txt:33]:
```suggestion
- A MongoDB server that you have  :manual:`installed locally</installation/>`, or
```

File: source/index.txt [hunk 2]
```diff
@@ -24,6 +24,17 @@ exploring your data in a visual environment, you can also use
...
+Prerequisites
+-------------
+
+To use |compass-short|, you must connect to an existing MongoDB 
+database. You can connect to:
+
+- A MongoDB server that you :manual:`install </installation/>` and 
+  manage, or
+
+- `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`__ database-as-a-service.
```
Comment (andf-mongodb) [inline source/index.txt:36]:
```suggestion
- A MongoDB Atlas cluster. `Atlas<https://www.mongodb.com/cloud/atlas?tck=docs_compass>`__ is our cloud-based MongoDB database-as-a-service.
```

Or something similar. Not liking the two-sentence approach, myself, but couldn't hit upon anything cleaner. Feel free to use / adjust / discard!

Comment (zach-carr) [reply source/index.txt:36]: Went with just:

- A MongoDB Atlas cluster.

There's a note right below that explains and links to Atlas - I was iffy on the original for that reason, I think it covers the would-be second sentence. Let me know if you think we should ignore the note.

Comment (andf-mongodb) [reply source/index.txt:36]: Perfect!

**Pull request merged**